packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -590,8 +590,10 @@ class TechnologyController {
 			'technologyCosts.costs',
 		]);
 
-		if (technology.status === technologyStatuses.PUBLISHED) {
+		if (technology.active) {
```

Comment:
The same should happen when the technology is published or not, right?

Reply:
Sim, mas nessa rota não cabe essa verificação